viite-UI/src/view/ProjectLinkLayer.js

```diff
@@ -631,7 +631,7 @@
         });
         var style = new ol.style.Style({
           stroke: new ol.style.Stroke({color: '#c6c00f', width: 13, lineCap: 'round'}),
-          zIndex: 11
+          zIndex: -1
```

Comment:
both terminated and its marker "C" have zIndex = 11. Dunno if one change then the other one should also change. 
Still this change seems visually good since terminated marker will be visible (e.g. in top of suravage links)